### ISSUE TITLE
GH Action: Run in ODK

### ISSUE THREAD

**Issue by joeflack4** [monarch-initiative/omim#97]
## Overview
This will ensure dependencies are consistent w/ ODK and will solve #92.

**Comment by joeflack4** [comment]
Not sure if this is feasible. If not, close.

**Comment by matentzn** [comment]
100% we have 1000 GH actions running on ODK, e.g. 

https://github.com/obophenotype/bio-attribute-ontology/blob/master/.github/workflows/qc.yml#L22

(the latest version is 1.4.1)

**Comment by joeflack4** [comment]
@matentzn I went ahead and opened a PR with initial attempt (#100). I got stuck here:
- https://github.com/INCATools/ontology-development-kit/issues/888

Ideally I'd like to not change the whole directory structure of the repository to use ODK as more of a software / package management environment. I can do that if it is necessary, but IMO simply including a local `robot.jar` seems like the much less complex solution in that case.

Also, there are [some docs](https://github.com/INCATools/ontology-development-kit#migrating-an-existing-ontology-repo-to-the-odk) that are a little helpful and give me ideas, but I think the case we want here is simpler / should ideally not require all these changes. However on that note, I should say that even for standard ODK use cases, I think these docs could be better. I think there should be an ordered list of exact steps to follow to set up a new repo or migrate an existing one.
